### PR TITLE
Feature/regarch 116 object detail tweaks

### PR DESCRIPTION
**Pull request by igor-kamil**
https://jira.sng.sk/browse/REGARCH-116
+ multiple ad-hoc fixes/tweaks during meeting with Luba

Screenshot 2020-09-09 at 14 07 07
Screenshot 2020-09-09 at 14 07 16
Screenshot 2020-09-09 at 14 07 37

### CODE REVIEW

File: resources/sass/components/_header.scss [hunk 1]
```diff
@@ -13,21 +13,22 @@
     }
 }
 
-#header {
+#header,  {
```
Comment (eronisko) [inline resources/sass/components/_header.scss:16]: Did you mean to leave in the comma?

File: app/Models/Building.php [hunk 2]
```diff
@@ -289,6 +289,11 @@ public static function getFilterValues($payload)
         ];
     }
 
+    public function has3DModel()
+    {
+        return true; //@TODO
```
Comment (eronisko) [inline app/Models/Building.php:294]: Do we have a JIRA issue for this?

File: resources/views/architects/show.blade.php [hunk 3]
```diff
@@ -6,24 +6,22 @@
...
         <div class="col-6 col-lg-3">
-            <div class="row">
-                <div class="col-10 col-sm-6 col-lg-8 col-xl-5">
+            <div class="border p-4 text-center w-100 vh-38">
                 @if($architect->has_image)
-                {{ $architect->image_tag->attributes(['class' => 'mh-100 mw-100']) }}
+                    {{ $architect->image_tag->attributes(['class' => 'mh-100 mw-100']) }}
+                @else
+                    {{-- Placeholder image --}}
+                    <img src="{{ asset('images/no-image-architect.svg') }}" class="mh-100 mw-100" alt="{{ $architect->full_name }}">
```
Comment (eronisko) [inline resources/views/architects/show.blade.php:16]: 🙌

**Review by eronisko — APPROVED** [review]

File: app/Models/Building.php (hunk 2 above)
Comment (igor-kamil) [reply app/Models/Building.php:294]: good point! no ...I don't even know how to get that data...

**Pull request merged**